Abhi-AnimeX/app/src/main/java/com/abhijith/animex/data/network/model/AnimeListResponse.kt

```diff
@@ -0,0 +1,375 @@
+package com.abhijith.animex.data.network.model
+
+import com.google.gson.annotations.SerializedName
+
+data class AnimeListResponse(
+
+    @SerializedName("pagination")
+    val pagination: Pagination? = null,
+
+    @SerializedName("data")
+    val data: List<AnimeDto?>? = null // can be null in case we reach the end of data
```

Comment:
default value can be an empty list?
WDYT

maybe null is more easy check than empty list, ok